.gitlab-ci.yml

```diff
@@ -40,6 +40,8 @@ default:
   id_tokens:
     SITE_ID_TOKEN:
       aud: https://asc-git.lanl.gov
+    CI_JOB_JWT:
```

Comment:
@rbberger is this change still needed?

Reply:
I have no other pipeline or project that needed that change.

Reply:
That was the change  that the CI was asking for on other projects.

Reply:
I'm confused---do we need this change or no?

Reply:
No. I don't think so. My theory is that re-git switched over to the new version (the thing that they were warning for many months), which deprecated `CI_JOB_JWT` and that there was some outdated runners still floating around while me and Karen were out. The outdated runner would then complain about not getting the no longer provided token. All it needed was to get that runner updated to accept the new SITE_ID_TOKEN, which seems to have eventually happened.